.github/workflows/tests.yaml

```diff
@@ -183,46 +186,3 @@ jobs:
 
       - name: Test
         run: yarn test --since origin/master
-
-  tests-legacy:
```

Comment:
any reason we are dropping node v12 support in this PR?

Reply:
I was thinking since node 12 reached its end of life in April of 2022 we should not support it

Reply:
Sounds good!